linetools/analysis/interactive_plot.py

```diff
@@ -353,7 +369,14 @@ def plotinit(self):
         a1.set_ylim(-4, 4)
         a0.axhline(0, color='0.7')
 
+        # Initial plot limits
         i0,i1 = self.indices
+        xmin = wa[i0]
+        xmax = wa[i1]
+        self.nav_dict = dict(nav=ltgu.navigate(0, 0, init=True))
+        self.nav_dict['xmnx'] = [xmin, xmax]
```

Comment:
I would find it much clearer to call this `x_minmax` instead of `xmnx`. Similarly for `ymnx`.  If you agree, guis.utils and guis.specwidgets also need changing.